### PR TITLE
Feature/skillcape emotes

### PR DESCRIPTION
**Pull request by SchauweM**
Added skillcape emote handling & a new getter in Player called getItemInEquipmentSlot

### CODE REVIEW

**Review by TheBlackParade — DISMISSED** [review]
:shipit:

**Comment by SchauweM** [comment]
Help needed: Emote sprite doesn't "light up" when a skillcape is worn

**Comment by SchauweM** [comment]
All issues were fixed in the latest push

File: src/plugins/buttons/player-emotes-plugin.ts [hunk 1]
```diff
@@ -117,9 +147,15 @@ export const action: buttonAction = (details) => {
     const { player, buttonId } = details;
 
     const emote = emotes[buttonId];
-
+    
     if(emote.name === 'SKILLCAPE') {
-        player.sendMessage(`You need to be wearing a skillcape in order to perform that emote.`);
+        if (skillcapeEmotes.some(item => item.itemIds.includes(player.getItemInEquipmentSlot('BACK')?.itemId))) {
```
Comment (Promises) [inline src/plugins/buttons/player-emotes-plugin.ts]: not really important, but a check if wearing any cape first might be slightly faster?

File: src/plugins/items/capes/skillcape-emotes.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { lockEmote, unlockEmote } from '@server/plugins/buttons/player-emotes-plugin';
+import { ActionType, RunePlugin } from '@server/plugins/plugin';
+import { equipAction } from '@server/world/actor/player/action/equip-action';
+
+export const skillcapes: Array<number> = [
```
Comment (Promises) [inline src/plugins/items/capes/skillcape-emotes.ts]: Also not really required but could you move these to itemids?

Comment (SchauweM) [reply src/plugins/items/capes/skillcape-emotes.ts]: I'll add them when I get off work later today!

Comment (SchauweM) [reply src/plugins/items/capes/skillcape-emotes.ts]: Finally found the time to fix the issue

File: src/plugins/buttons/player-emotes-plugin.ts (hunk 1 above)
Comment (SchauweM) [reply src/plugins/buttons/player-emotes-plugin.ts]: Added a check as requested

**Comment by SchauweM** [comment]
Ready for review.

File: src/world/actor/actor.ts [hunk 3]
```diff
@@ -174,6 +174,11 @@ export abstract class Actor {
         this.updateFlags.animation = animation;
     }
 
+    public stopAnimation(): void {
```
Comment (TheBlackParade) [inline src/world/actor/actor.ts:177]: 💯

File: src/world/actor/actor.ts [hunk 4]
```diff
@@ -182,6 +187,11 @@ export abstract class Actor {
         this.updateFlags.graphics = graphics;
     }
 
+    public stopGraphics(): void {
```
Comment (TheBlackParade) [inline src/world/actor/actor.ts:190]: 💯

**Review by TheBlackParade — DISMISSED** [review]
:shipit:

**Review by TheBlackParade — APPROVED** [review]

**Pull request merged**